src/common/nlmeans_core.c

```diff
@@ -55,7 +51,9 @@
 // recomputing for both scalar and SSE on a Threadripper due to increased memory writes; this may differ on
 // architectures with slower multiplication.
 //#define CACHE_PIXDIFFS
-//#define CACHE_PIXDIFFS_SSE
+
+// re-checked SSE2 codepath performance on 8-core Intel (tested with nthreads 4, 8, 16) there was no gain
```

Comment:
I'm not sure we need such comment in the code. In some years a dev reading that will ask himself, what is this about? Is there some SSE support somewhere?... For sure this must be in the comment log.